server/games/game.py

```diff
@@ -457,48 +511,61 @@ def persist_rating_change_stats(self, rating_groups, rating='global'):
 
         rating_table = '{}_rating'.format('ladder1v1' if rating == 'ladder' else rating)
 
-        with (yield from db.db_pool) as conn:
-            cursor = yield from conn.cursor()
+        async with db.db_pool.get() as conn:
+            cursor = await conn.cursor()
 
             for player, new_rating in new_ratings.items():
                 self._logger.debug("New %s rating for %s: %s", rating, player, new_rating)
                 setattr(player, '{}_rating'.format(rating), new_rating)
-                yield from cursor.execute("UPDATE game_player_stats "
-                                          "SET after_mean = %s, after_deviation = %s, scoreTime = NOW() "
-                                          "WHERE gameId = %s AND playerId = %s",
-                                          (new_rating.mu, new_rating.sigma, self.id, player.id))
+                await cursor.execute("UPDATE game_player_stats "
+                                     "SET after_mean = %s, after_deviation = %s, scoreTime = NOW() "
+                                     "WHERE gameId = %s AND playerId = %s",
+                                     (new_rating.mu, new_rating.sigma, self.id, player.id))
                 if rating != 'ladder':
                     player.numGames += 1
 
-                yield from cursor.execute("UPDATE " + rating_table + " "
-                                          "SET mean = %s, is_active=1, deviation = %s, numGames = numGames + 1 "
-                                          "WHERE id = %s".format(rating), (new_rating.mu, new_rating.sigma, player.id))
+                await self._update_rating_table(cursor, rating_table, player, new_rating)
+
                 self.game_service.player_service.mark_dirty(player)
 
-    def set_player_option(self, id, key, value):
+    async def _update_rating_table(self, cursor, table: str, player: Player, new_rating):
+        # If we are updating the ladder1v1_rating table then we also need to update
+        # the `winGames` column which doesn't exist on the global_rating table
+        if table == 'ladder1v1_rating':
+            is_victory = self.outcome(player) == GameOutcome.VICTORY
+            await cursor.execute(
+                "UPDATE ladder1v1_rating "
+                "SET mean = %s, is_active=1, deviation = %s, numGames = numGames + 1, winGames = winGames + %s "
+                "WHERE id = %s", (new_rating.mu, new_rating.sigma, 1 if is_victory else 0, player.id))
+        else:
+            await cursor.execute(
+                "UPDATE " + table + " "
+                "SET mean = %s, is_active=1, deviation = %s, numGames = numGames + 1 "
+                "WHERE id = %s", (new_rating.mu, new_rating.sigma, player.id))
```

Comment:
That would be better, however, I was asked to do queries this way because when the logging cursor prints the multiline strings it keeps the formatting and it looks like an aweful mess. 

I've addressed this issue in: https://github.com/FAForever/server/pull/397